src/data/nat/basic.lean

```diff
@@ -260,6 +260,21 @@ iff.intro eq_zero_of_mul_eq_zero (by simp [or_imp_distrib] {contextual := tt})
 @[simp] protected theorem zero_eq_mul {a b : ℕ} : 0 = a * b ↔ a = 0 ∨ b = 0 :=
 by rw [eq_comm, nat.mul_eq_zero]
 
+lemma le_mul_pos_right {m n : ℕ} (h : n > 0) : m ≤ m * n :=
+begin
+rw [← mul_one(m),mul_assoc,one_mul],
+exact mul_le_mul_of_nonneg_left (nat.succ_le_of_lt h) dec_trivial,
+end
+
+lemma le_mul_pos_left {m n : ℕ} (h : n > 0) : m ≤ n * m :=
+begin
+rw [← one_mul(m),←mul_assoc,mul_one],
+exact mul_le_mul_of_nonneg_right (nat.succ_le_of_lt h) dec_trivial,
+end
+
+lemma le_mul_ge_one (m n : ℕ) (h : n ≥ 1) : m ≤ m * n :=
```

Comment:
Notice that `le_mul_pos_right h` also works as a proof! What's going on here is that `n ≥ 1` is actually definitionally equal to `n > 0`.

I'd suggest removing this lemma.

Reply:
Ah ok did not realise they are actually perfectly equivalent. I had only left that form in since that was specifically what you requested and I had the others already.

Reply:
Actually, I think this lemma should be generalized to an ordered semiring or something, it's not a nat-specific theorem.